pages/dkp/kommander/1.4/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: layout.pug
+navigationTitle: Requirements for Attaching an Existing Cluster to Kommander
+title: Requirements for Attaching an Existing Cluster to Kommander
+menuWeight: 3
+excerpt: Requirements for attaching an existing cluster to Kommander
+beta: true
+---
+
+## Basic requirements
+
+You can attach existing clusters to Kommander whether or not those clusters have networking restrictions.
+
+For attaching existing clusters without networking restrictions, the requirements depend on the Kommander 1.X version that you plan to use. The supported Kubernetes versions are between version 1.16.x and 1.20.x with a maximum version of 1.20.5. To attach an existing cluster, the Kommander management cluster must be able to reach the services and the api-server of the target cluster.
```

Comment:
This needs elaboration, it is not at all clear what the connection between 'which Kommander 1.X version' I am using and the Kubernetes version. 

Also, the last clause, i.e.: "the Kommander management cluster must be able to reach the services and the api-server of the target cluster." really has nothing to do with the rest of this paragraph - this is a general requirement, not specific to 'existing clusters without networking restrictions'.

```suggestion
For attaching existing clusters without networking restrictions, the requirements depend which Kommander 1.X version you are using. Each version of Kommander supports a specific range of Kubernetes versions; you must ensure that the target cluster is running a compatible version.

For example, Kommander 1.3 supports Kubernetes versions between 1.17.0 and 1.19.x, so any cluster you wish to attach using Kommander 1.3 must be running a version between these versions." 
```

Reply:
Do we need to say which "Kommander 1.x version" - or can we just say "depends on which Kommander version"? This seems pretty refined, when it doesn't need to be for this sentence in my opinion. The specific versions show up later.

Reply:
Fixed as noted.